docs/aserto-console/directory/add-group.mdx

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: Adding Groups
+title: Adding Groups
+description: Adding a group to the directory
+---
+
+# Adding Groups
+
+A Group is a built-in object type. To add a new object instance of the type "group", click on the "Group" sub-menu, which shows all the existing groups.
```

Comment:
Q: is a group an object or subject type? it's listed as a subject type for me:
![image](https://user-images.githubusercontent.com/11022003/201787345-1bb4352a-ee60-4ba1-a1f2-50e8bbddd06b.png)

but the screenshot in these docs says object type
![image](https://user-images.githubusercontent.com/11022003/201787435-dfb8e82e-dfc6-485d-87ce-6e4815d4ba99.png)

Reply:
I noticed this too. Turns out that new accounts have "Group" created as an object, not a subject. There is a PR in the works to fix it.

Reply:
When we get the right types back, I'll create a new PR that has the right screenshots